### ISSUE TITLE
[Keyboard navigation - Make code Arcade- Project-editor]:  Keyboard focus moving incorrect order after the 'Go back' control.

### ISSUE THREAD

**Issue by BURRAshiva01** [microsoft/pxt-microbit#6117]
### User Experience: 
Keyboard dependent people will get impacted if keyboard focus moves incorrect order while navigating using tab key. Due to which the end users will not be able to distinguish between the controls present on the page and can find difficulty in accessing the page.

**Note:** Login Credentials must not be provided within the bug.

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/#editor
2. Now 'editor' page will open and navigate to the 'Git hub' control and select it.
3. Navigate to the 'Pull changes' button and observe the issue. 

### Actual Result:
Keyboard focus moving incorrect order after the 'Go back' control.

**Observations:** After the 'Go back' button, keyboard focus moving to the 'Pull Changes' control instead of 'Open repository in GitHub' link. 

### Expected Result:
Keyboard focus should move in logical or sequential order.

**Example:** Keyboard focus should sequential order i.e Go back' Control->'Open repository in GitHub' link->'Pull Changes' control.

### MAS Reference:
[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance/previous test pass bug id's:** N/A

**UIA/DOM details available:** 

Yes (Refer the attachment)
### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:
**OS Version:** Windows 11 24H2 (OS Build:26100.3037)
**Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
**Test Matrix:** Chrome + NVDA
**URL:** https://www.microsoft.com/en-us/makecode/teach

### Attachment:

<img width="987" alt="Image" src="https://github.com/user-attachments/assets/81bfd661-95fb-4a31-9f18-3681ac7d1ce5" />

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue is still repro. Please find attachment for the reference

<img width="959" alt="Image" src="https://github.com/user-attachments/assets/43c7d398-82b2-46c7-890f-e295170111e4" />
